packages/pg/lib/index.js

```diff
@@ -23,6 +24,8 @@ var PG = function (clientConstructor) {
   this.Connection = Connection
   this.types = require('pg-types')
   this.DatabaseError = DatabaseError
+  this.escapeIdentifier = escapeIdentifier
+  this.escapeLiteral = escapeLiteral
```

Comment:
This is just new though, right? Why?

Reply:
This is new yes, as for why [your previous comment](https://github.com/brianc/node-postgres/pull/2954#pullrequestreview-1378415905) suggested that the escape* methods be exported on pg, this adds them as exports to the pg object. On my end it looks like the github diff viewer annoyingly cuts off the part where it shows these belonging to PG, but looking at the [full file](https://github.com/brianc/node-postgres/blob/fe01e3db7f578039a9f4a1daa4cc3d5816c4cb2e/packages/pg/lib/index.js#L18-L29) it shows the rest of the context.

Is there an issue with exporting them this way?